package.json

```diff
@@ -7,40 +7,41 @@
     "test": "test"
   },
   "dependencies": {
-    "bower": "~1.3.12",
+    "bower": "~1.8.0",
     "bower-endpoint-parser": "~0.2.2",
-    "chalk": "~1.0.0",
-    "debug": "~2.1.3",
-    "download": "git://github.com/adaptlearning/download.git",
-    "grunt": "~0.4.5",
-    "lodash": "~3.6.0",
-    "mkdirp": "^0.5.1",
-    "nodegit": "^0.6.3",
-    "npm": "~2.7.4",
-    "prompt": "~0.2.14",
-    "q": "^1.2.1",
-    "q-io": "~1.12.0",
-    "semver": "^4.3.3",
-    "speakingurl": "~1.1.0",
-    "url-join": "0.0.1",
-    "uuid": "~2.0.1"
+    "chalk": "~1.1.3",
+    "debug": "~2.3.0",
+    "download": "~5.0.2",
+    "grunt": "~1.0.1",
+    "lodash": "~4.16.6",
+    "mkdirp": "~0.5.1",
+    "nodegit": "~0.16.0",
+    "npm": "~3.10.9",
+    "prompt": "~1.0.0",
+    "q": "~1.4.1",
+    "q-io": "~1.13.2",
+    "semver": "~5.3.0",
+    "speakingurl": "~10.0.0",
+    "url-join": "~1.1.0",
+    "uuid": "~2.0.3"
   },
   "devDependencies": {
     "expect.js": "~0.3.1",
-    "grunt": "~0.4.5",
-    "grunt-mocha-test": "~0.12.7",
-    "matchdep": "~0.3.0",
-    "mocha": "~2.2.1",
-    "mockery": "~1.4.0",
-    "rimraf": "~2.3.2",
-    "sinon": "~1.14.1"
+    "grunt": "~1.0.1",
```

Comment:
I think we can safely remove this as `grunt` already exists in dependencies object.